sdk-python/omnilogger/openai_listener.py

```diff
@@ -0,0 +1,13 @@
+import openai
+
+
+class CustomCompletion(openai.Completion):
+    @classmethod
+    def create(cls, model: str, prompt: str, **kwargs):
+        result = super().create(model=model, prompt=prompt, **kwargs)
+        print(prompt, result)  # TODO: reformat and call send_to_db
+        return result
+
+
+def start_openai_listener():
+    openai.Completion = CustomCompletion
```

Comment:
that's really neat! monkey patching 🐵

Reply:
It looks like the `Completion` API has [been marked as legacy now](https://platform.openai.com/docs/guides/gpt/completions-api). Should we be patching the [`ChatCompletions` API instead?](https://platform.openai.com/docs/guides/gpt/chat-completions-api)

Reply:
Was the usage of the `Completion` API intentional? Was there a reason it was implemented over other APIs?

I agree about the ticket!

Reply:
Using `Completion` is a legacy from [the PoC](https://github.com/Theodo-UK/OmniLog/pull/1/files)... I don't think it was intentional (@juskek?)

Reply:
It was not intentional, just took the example off the OpenAI docs